environment.yml

```diff
@@ -17,3 +17,5 @@ dependencies:
   - pytest
   - psutil
   - gxx<12
+  - pip:
```

Comment:
AFAIK this is a dangerous idea. Note that this project you link to is a short and single MIT-licensed class: https://github.com/fadel/pytorch_ema/blob/master/torch_ema/ema.py

I would consider just including it in the repo with the appropriate copyright notice.